From 8313c00637730d6d941da14079d4f516d8691625 Mon Sep 17 00:00:00 2001
From: Christian Bruel <christian.bruel@foss.st.com>
Date: Mon, 10 Mar 2025 16:42:09 +0100
Subject: [PATCH] dt-bindings: stm32: document stm32mp215f-dk-aarch32

Add new entry for stm32mp215f-dk built for aarch32

Signed-off-by: Christian Bruel <christian.bruel@foss.st.com>
Change-Id: Ib0b7506369a623891acf72f07ea66465152c4515
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/443277
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
---
 Documentation/devicetree/bindings/arm/stm32/stm32.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/arm/stm32/stm32.yaml b/Documentation/devicetree/bindings/arm/stm32/stm32.yaml
index 9ce5e3302c8f..808036450f50 100644
--- a/Documentation/devicetree/bindings/arm/stm32/stm32.yaml
+++ b/Documentation/devicetree/bindings/arm/stm32/stm32.yaml
@@ -189,6 +189,7 @@ properties:
         items:
           - enum:
               - st,stm32mp215f-dk
+              - st,stm32mp215f-dk-aarch32
           - const: st,stm32mp215
 
 additionalProperties: true
-- 
2.50.1

